gitbootcamp.rst

```diff
@@ -28,26 +28,26 @@ relevant to CPython's workflow.
 Forking CPython GitHub Repository
 ---------------------------------
 
-You'll only need to do this once.
+You will only need to do this once.
 
 1. Go to https://github.com/python/cpython.
 
 2. Press ``Fork`` on the top right.
 
 3. When asked where to fork the repository, choose to fork it to your username.
 
-4. Your fork will be created at https://github.com/<username>/cpython.
+4. Your forked CPython repository will be created at https://github.com/<username>/cpython.
 
 .. _clone-your-fork:
 
-Cloning The Forked CPython Repository
--------------------------------------
+Cloning a Forked CPython Repository
+--------------------------------------
```

Comment:
```suggestion
-----------------------------------
```

Reply:
Good catch, the dashes in the title need to have the same character count as the text of the title.

Reply:
Thanks @eamanu.